### DISCUSSION TITLE
Meshcmd not work in xp32bit

### DISCUSSION THREAD

**Discussion by mohammmedmellal** [Ylianst/MeshCentral#5168] [Q&A]
Hello everyone, I am a neophyte :) I have a fleet of PCs (hp, dell, lenovo) in the field of industry, Meshcemd works well for me under win10 in 64bit (ltsc) but not on xp 32 bit (sp3 version 2002), I have the error message "is not a valid win32 application" thank you for your support !

**Comment by si458** [comment]
This sounds like you are trying to use the 64bit version instead of the 32bit version
When you download it from the Web panel the should be an option to change it to 32bit

**Reply by mohammmedmellal** [reply]
Thanks si458 for yr quick answer, yes i dwnld 32bit ver. from website and also from meshcentral installed localy...but don't work on my xp.. even after desabling antivirus SEP and EDR FireEye ! i also try tool in intel_sdk named "Host_Based_Setup_Module" but same rsult can't run it... my gaol is to remotely activate AMT in CCM mode, thanks for your help :)

**Reply by si458** [reply]
I'm not sure I'm afraid, normally the invalid application error means it's trying to run a 64bit exe on a 32bit computer, I've never gotten amt to work so I don't use it sorry

**Reply by si458** [reply]
have you tried the exes from this website- https://www.meshcommander.com/meshcommander/meshcmd ?

**Reply by mohammmedmellal** [reply]
YES, i downlod 32bit exe from there , but still not working ...

**Reply by mohammmedmellal** [reply]
other info  : i have already remotely activate AMT on 120 pc with this commandes : 
1st/ PsExec.exe -s \\pcname  cmd
2nd/ Meshcmd amtccm --password Password

**Comment by Ylianst** [comment]
Hi. So, MeshCMD is actually the MeshAgent executable with added JavaScript appended to the executable to make it act like a command line tool. The MeshAgent does not support Windows XP. The main problem was thread and process handling which is super old and difficult to support. So, there is no plan to ever have the MeshAgent support Windows XP (32 or 64bit).

Since MeshCMD is used in this case for Intel AMT activation on Windows XP, I would contact Intel support and ask what options they offer. I would think they will not have anything, but it's worth a try.

**Reply by mohammmedmellal** [reply]
thanks a lot Ylianst for yr answer, let me know if there is an other "simple" methode for remotely activate amt on those XP's 32bits PC (java script, by installing agent, executing command from serevr...) . Thnx

**Comment by silversword411** [comment]
I'll never understand the efforts people will go to install old unsupported and known vulnerable software and hardware into their networks. #HackMePlease
